python2.7 windows wheels failed to build for 0.19.0

I'll have to figure out a way to fix that

Here's the error:

```
DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 won't be maintained after that date. A future version of pip will drop support for Python 2.7.
Processing d:\a\1\s
ERROR: Exception:
Traceback (most recent call last):
  File "d:\a\1\s\.tox\py27\lib\site-packages\pip\_internal\cli\base_command.py", line 178, in main
    status = self.run(options, args)
  File "d:\a\1\s\.tox\py27\lib\site-packages\pip\_internal\commands\wheel.py", line 164, in run
    resolver.resolve(requirement_set)
  File "d:\a\1\s\.tox\py27\lib\site-packages\pip\_internal\resolve.py", line 131, in resolve
    self._resolve_one(requirement_set, req)
  File "d:\a\1\s\.tox\py27\lib\site-packages\pip\_internal\resolve.py", line 294, in _resolve_one
    abstract_dist = self._get_abstract_dist_for(req_to_install)
  File "d:\a\1\s\.tox\py27\lib\site-packages\pip\_internal\resolve.py", line 242, in _get_abstract_dist_for
    self.require_hashes
  File "d:\a\1\s\.tox\py27\lib\site-packages\pip\_internal\operations\prepare.py", line 347, in prepare_linked_requirement
    progress_bar=self.progress_bar
  File "d:\a\1\s\.tox\py27\lib\site-packages\pip\_internal\download.py", line 873, in unpack_url
    unpack_file_url(link, location, download_dir, hashes=hashes)
  File "d:\a\1\s\.tox\py27\lib\site-packages\pip\_internal\download.py", line 778, in unpack_file_url
    shutil.copytree(link_path, location, symlinks=True)
  File "c:\hostedtoolcache\windows\python\2.7.14\x86\Lib\shutil.py", line 222, in copytree
    raise Error, errors
Error: [('D:\\a\\1\\s\\libsass\\test\\e2e\\unicode-pwd\\Sa\xb4ss-UT\xb8F8', 'c:\\users\\vssadm~1\\appdata\\local\\temp\\pip-req-build-e7ok_o\\libsass\\test\\e2e\\unicode-pwd\\Sa\xb4ss-UT\xb8F8', "[Errno 2] No such file or directory: 'D:\\\\a\\\\1\\\\s\\\\libsass\\\\test\\\\e2e\\\\unicode-pwd\\\\Sa\\xb4ss-UT\\xb8F8'"), ('D:\\a\\1\\s\\libsass\\test\\e2e\\unicode-pwd\\S\xe1ss-UTF8', 'c:\\users\\vssadm~1\\appdata\\local\\temp\\pip-req-build-e7ok_o\\libsass\\test\\e2e\\unicode-pwd\\S\xe1ss-UTF8', "[Errno 2] No such file or directory: 'D:\\\\a\\\\1\\\\s\\\\libsass\\\\test\\\\e2e\\\\unicode-pwd\\\\S\\xe1ss-UTF8'")]
##[error]Bash exited with code '2'.
```

--- a/azure-pipelines.yml
+++ b/azure-pipelines.yml
@@ -19,12 +19,29 @@ jobs:
 - template: job--pre-commit.yml@asottile
 - template: job--python-tox.yml@asottile
   parameters:
-    toxenvs: [py27, py35, py36, py37]
+    toxenvs: [py27]
+    os: windows
+    name_postfix: _py27
+    wheel_tags: true
+    pre_test:
+    - script: rm -rf libsass/test
+- template: job--python-tox.yml@asottile
+  parameters:
+    toxenvs: [py27]
+    os: windows
+    architecture: x86
+    name_postfix: _py27_x86
+    wheel_tags: true
+    pre_test:
+    - script: rm -rf libsass/test
+- template: job--python-tox.yml@asottile
+  parameters:
+    toxenvs: [py35, py36, py37]
     os: windows
     wheel_tags: true
 - template: job--python-tox.yml@asottile
   parameters:
-    toxenvs: [py27, py35, py36, py37]
+    toxenvs: [py35, py36, py37]
     os: windows
     architecture: x86
     name_postfix: _x86

